src/app/areas/area.service.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { async, inject, TestBed } from '@angular/core/testing';
+import { BaseRequestOptions, Http, Response, ResponseOptions } from '@angular/http';
+import { MockBackend } from '@angular/http/testing';
+
+import { cloneDeep } from 'lodash';
+
+import { AuthenticationService, UserService, AUTH_API_URL } from 'ngx-login-client';
+import { Broadcaster, Logger } from 'ngx-base';
+
+import { WIT_API_URL } from "../api/wit-api";
+import { Area } from '../models/area';
+import { AreaService } from './area.service';
+
+describe('Service: AreaService', () => {
+
+  let areaService: AreaService;
+  let mockService: MockBackend;
+  let fakeAuthService: any;
+
+  beforeEach(() => {
+    fakeAuthService = {
+      getToken: function () {
+        return '';
+      },
+      isLoggedIn: function () {
+        return true;
+      }
+    };
+    TestBed.configureTestingModule({
+      providers: [
+        Logger,
+        BaseRequestOptions,
+        MockBackend,
+        {
+          provide: Http,
+          useFactory: (backend: MockBackend,
+                       options: BaseRequestOptions) => new Http(backend, options),
+          deps: [MockBackend, BaseRequestOptions]
+        },
+        {
+          provide: AuthenticationService,
+          useValue: fakeAuthService
+        },
+        AreaService,
+        {
+          provide: WIT_API_URL,
+          useValue: "http://example.com"
+        },
+        {
+          provide: AUTH_API_URL,
+          useValue: 'http://example.com/auth'
+        },
+        UserService,
+        Broadcaster
+      ]
+    });
+  });
+
+  beforeEach(inject(
+    [AreaService, MockBackend],
+    (service: AreaService, mock: MockBackend) => {
+      areaService = service;
+      mockService = mock;
+    }
+  ));
+
+  let responseData: Area[] = [
```

Comment:
This does not match the Area type.
It is missing:
`
        'parent_path': null,
        'parent_path_resolved': null,
`